applications/advanced-concepts/tags-and-placeholders.md

```diff
@@ -0,0 +1,17 @@
+---
+description: Tags and placeholders in machine translation
+---
+
+# Tags and placeholders
+
+**Tags and placeholders** are pieces of non-translatable code. While tags format and structure texts, placeholders are variables for dynamic content.
```

Comment:
I'd break this up.

> **Tags** and **placeholders** are pieces of non-translatable code inside the input text.

> ### Tags
> Tags format, structure and annotate text.  Tags can also be used as placeholders.

> Tags are usually XML, for example HTML.

> Most online machine translation systems launched with support for tags, because they were for translating HTML pages.

> There are two approaches to translating tags:


> ## Placeholders

Reply:
One key thing to note is that placeholder schemes are not actually supported by the major machine translation APIs.

But need to say it in a timeless way, since it could change.